androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -230,6 +261,16 @@ fun EnhancedVideoPlayer(
                             ExoPlayer.REPEAT_MODE_OFF
                         }
                     },
+                    selectedQualityTrack = {
+                        selectedTrack
+                    },
+                    qualityTracks = {
+                        trackQualityOptions
+                    },
+                    onQualityChanged = { selectedQualityTrack ->
+                        selectedTrack = selectedQualityTrack
+                        exoPlayer.setVideoQuality(selectedQualityTrack)
```

Comment:
This `setVideoQuality` uses `setTrackSelectionParameters` method that has the following comments:
```
Unsupported parameters will be silently ignored.
...
This method must only be called if {@link #COMMAND_SET_TRACK_SELECTION_PARAMETERS} is
   * {@linkplain #getAvailableCommands() available}
```
Maybe it is better to use the `onTrackSelectionParametersChanged` method  to set `selectedTrack` with the real value from exoplayer.

Reply:
Added the suggestion! Ty